ui/pages/confirmations/hooks/alerts/useDomainMismatchAlert.ts

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from 'react';
+import {
+  isValidSIWEOrigin,
+  WrappedSIWERequest
+} from '@metamask/controller-utils';
+
+import { Alert } from '../../../../ducks/confirm-alerts/confirm-alerts';
+import { Severity } from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { SignatureRequestType } from '../../types/confirm';
+import { isSIWESignatureRequest } from '../../utils';
+import useCurrentConfirmation from '../useCurrentConfirmation';
+
+const useDomainMismatchAlerts = (): Alert[] => {
```

Comment:
Minor, but could this instead be `export function` so it's easier to auto-complete the name on import?

Alternatively, we could do `export default function` for simpler syntax?

Reply:
updated [ca059ab](https://github.com/MetaMask/metamask-extension/pull/25281/commits/ca059ab87e5395ca5a56692b03386fcab11f6ea4)